### ISSUE TITLE
May I ask what is the difference between v1 and v2, especially in the field of adversarial training

### ISSUE THREAD

**Issue by MetaInsight7** [jiequancui/DKL#3]
May I ask what is the difference between v1 and v2, especially in the field of adversarial training

**Comment by jiequancui** [comment]
Hi, thanks for your interest in our work.

IKL loss in the v1 version of our NeurIPS paper is a special case of the GKL loss in the v2 version (the paper will be released soon).
Hyper-parameter tunning can lead to further improvement.
We will release new models as soon as possible.

Best,
Jiequan

**Comment by MetaInsight7** [comment]
thank you！
